stable/traefik/values.yaml

```diff
@@ -34,6 +34,80 @@ acme:
   email: admin@example.com
   staging: true
   logging: false
+  challengeType: "http-01"  # http-01 or dns-01
```

Comment:
Isn't the default currently in use the TLS-SNI-01 challenge? The PR that adds support for the HTTP-01 challenge has not been merged yet.

Reply:
I realize now that TLS-SNI-01 has been deprecated by Let's Encrypt. I understand why (vulnerabilities), but this is unfortunate. It nearly forces a breaking change on this chart.

For now, I'm going to suggest that we still treat TLS-SNI-01 as the default. (Traefik still supports it after all.) If it doesn't work, that's between the user and Let's Encrypt and isn't this chart's fault. (fwiw, if I understand correctly, it _will_ work for renewing existing certificates that were initially issued using that challenge.) i.e. We should, for now, treat this as "not our problem."

I think we can very soon consider a major revision of this chart and we can straighten this out better then.